pkg/cli/root.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package cli implements the commands for the Lumberjack CLI.
+package cli
+
+import (
+	"context"
+
+	"github.com/abcxyz/lumberjack/internal/version"
+	"github.com/abcxyz/pkg/cli"
+)
+
+// rootCmd defines the starting command structure.
+var rootCmd = func() cli.Command {
+	return &cli.RootCommand{
+		Name:    "ljctl",
```

Comment:
Hmm - we should probably name this `lumberjackctl`. `ljctl` is kinda weird.

Reply:
Cool, I am bad at naming, was thinking maybe shorter is preferred.

Reply:
I vote for "lumberctl" :) After all, we are looking at logs (lumber).

Reply:
Changed to the shorter version lumberctl, let me know if any concerns. Thanks both!

Reply:
@sethvargo friendly ping